src/Raven.Client/Documents/Indexes/ExpressionStringBuilder.cs

```diff
@@ -1760,8 +1760,21 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
             {
                 var type = node.Method.GetGenericArguments()[0];
                 _loadDocumentTypes.Add(type);
-                var collection = _conventions.GetCollectionName(type);
-                Out($", \"{collection}\"");
+
+                var parameters = node.Method.GetParameters();
+                if (parameters.Length == 1)
+                {
+                    var collection = _conventions.GetCollectionName(type);
```

Comment:
Do we need to do escaping here?

Reply:
I'm not sure I follow. Did you mean one line below:
`Out($", \"{collection}\"");`
? 
We have `\"` to pass collection name as string parameter - `LoadDocument("users/1", "Users")`

Reply:
Yes, what happens if the collection contains `"` ?

Reply:
If `FindCollectionName` convention was overwritten and returned collection name would include special characters then I think yes we would need to escape it here.

I opened for verify all usages: https://issues.hibernatingrhinos.com/issue/RavenDB-14681